modules/container-converter/assets/js/editor/commands/convert.js

```diff
@@ -0,0 +1,65 @@
+import CommandHistory from 'elementor-document/commands/base/command-history';
+import Migrator from '../migrator';
+
+export class Convert extends CommandHistory {
+	getHistory() {
+		return {
+			type: __( 'Converted to Container', 'elementor' ),
+			title: __( 'Section', 'elementor' ),
+		};
+	}
+
+	validateArgs( args = {} ) {
+		this.requireContainer( args );
+	}
+
+	apply( args ) {
+		this.constructor.convert( args );
+	}
+
+	/**
+	 * Convert an element to Container.
+	 *
+	 * TODO: It's static in order to be able to test that without initializing the whole editor in a browser.
+	 *  Should be moved to `apply()` when there is a proper way to test commands using jest.
+	 *
+	 * @param {Container} container - Element to convert.
+	 * @param {Container} rootContainer - Root element to migrate the `container` into (used for recursion).
+	 *
+	 * @return {void}
+	 */
+	static convert( { container, rootContainer = container.parent } ) {
+		const { view, type: elType } = container;
+
+		// TODO: Maybe use `view._parent.collection.indexOf( this.model )`.
+		const at = view._index;
+
+		// Copy the element as is without converting.
+		if ( ! Migrator.canConvertToContainer( elType ) ) {
+			$e.run( 'document/elements/copy', { container } );
+			$e.run( 'document/elements/paste', { container: rootContainer, at } );
```

Comment:
Maybe use the `document/elements/create` also here, seems a little bit weird to use copy and paste, especially that if the user will later do 'paste' he will get the last element that was copied by you.

I mean if each command is a user action, the user did not want to do 'copy' and 'paste'.

Reply:
Totally right, done